core/tests/protractor/collections.js

```diff
@@ -30,6 +30,10 @@ describe('Collections', function() {
       '.protractor-test-reload-collection-button')).first().click();
     general.acceptAlert();
     browser.waitForAngular();
+    element.all(by.css(
```

Comment:
Yes, It'll be great. I've moved reloadAllExplorations() to admin.js but I think -- creating reloadCollection(index) for just a single collection (as of now) is not that efficient.